HW3_Daria_Dolia.py

```diff
@@ -0,0 +1,51 @@
+import random
+
+
+def get_valid_input():
+    while True:
+        try:
+            number_of_students = int(input('How many students --> '))
+            if number_of_students <= 0:
+                print('Sorry, no numbers below zero\n')
+            else:
+                return number_of_students
+        except ValueError:
+            print('Invalid input. Only integers.\nTry again\n')
+
+
+def create_students_dictionary(number_of_students):
+    students = {}
+    for i in range(number_of_students):
+        score = random.randint(50, 100)
+        verdict = random.choice(['Passed', 'Failed'])
+        students[f'Student {i+1}'] = (score, verdict)
+    return students
+
+
+def find_passing_score(students):
+    max_failing_score = 50
+    min_passing_score = 100
```

Comment:
I'd recommend to move these two values out of the function and to the file level. In Python constants are usually name with Capital letters only, like `MAX_FAILING_SCORE`